docs/conf/portland/2025/news/welcome-portland-2025.md

```diff
@@ -0,0 +1,112 @@
+---
+template: {{year}}/generic.html
+og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+---
+
+```{eval-rst}
+.. post:: Apr 28, 2025
+   :tags: {{shortcode}}-{{year}}
+```
+
+# Welcome to Write the Docs Portland 2025!
+
+We have officially arrived at conference week! We're excited to gather in Portland for our 13th year and our 3rd time at Revolution Hall. We hope you are as excited as we are! Below is an overview of the conference, as well as a couple of announcements.
+
+## Important Links
+
+The website is full of wonderful information about the conference, venue, and Portland. We encourage you to check out a few of our pages below.
+
+- [Attendee Guide](https://www.writethedocs.org/conf/portland/2025/attendee-guide/)
+- [Visiting Portland](https://www.writethedocs.org/conf/portland/2025/visiting/)
+- [Schedule](https://www.writethedocs.org/conf/portland/2025/schedule/)
+- [Code of Conduct](https://www.writethedocs.org/conf/portland/2025/code-of-conduct/)
+
+```{figure} /_static/conf/images/pics/portland-unconference.jpg
+```
+
+## How to Participate in the Conference
+
+- **Hike:** Join us on the hike on Saturday for our unofficial kick off to the conference. Hike is free, but space is limited. [Register with a ticket](https://ti.to/writethedocs/write-the-docs-portland-2025).
```

Comment:
Can we highlight this? Bunch of folks asking in slack

Reply:
We can link directly to the hike ticket like on the hike page as well.